server/channels/app/platform/web_conn.go

```diff
@@ -89,6 +90,7 @@ type WebConn struct {
 	Locale           string
 	Sequence         int64
 	UserId           string
+	PostedAck        bool
```

Comment:
Unless needed from the outside (unlikely) I'd make it unexported.

Reply:
Ah indeed. Fair enough, this is fine then given it's safe for concurrent access, no need for the method.